packages/docs-website/docs/protocol-tutorial/execute-state-transitions.md

```diff
@@ -107,9 +107,23 @@ As a state channel participant, it is advised to check the `FixedPart` of any ch
 
 - `chainId` -- This needs to match the id of the chain where assets are to be locked. The root of the funding tree for this channel.
 - `participants` -- This should have length at least 2, but no more than 255, and include a public key (account) that you control. Each entry should be a nonzero ethereum address.
-- `channelNonce` -- This should be different to any previous channelNonce used by the same `participants` and `chainId`. This is to prevent states from previous channels being "replayed" to conclude subsequent channels with unintended outcomes.
+- `channelNonce` -- This should be different to any previous channelNonce used by the same `participants` and `chainId`. This is to prevent states from previous channels being "replayed" to conclude subsequent channels with unintended outcomes. Must be less than `2**48-1`.
 - `appDefinition` -- There should be a [`ForceMoveApp`]('contract-api/natspec/ForceMove') compliant contract deployed at this address, and you should have confidence that it is not malicious or suffering from security flaws. You should inspect the source code (which should be publically available and verifiable) or appeal to a trusted authority to do this.
-- `challengeDuration` -- In the extreme, this should be at least 1 block time (15 seconds on mainnet) and less than `2**48-1` seconds. In practice we recommend somewhere between 5 minutes and 5 months.
+- `challengeDuration` -- In the extreme, this should be at least 1 block time (15 seconds on mainnet) and less than `2**48-1` seconds. Whatever it is set to, the channel should be closed long before `2**48 - 1 - challengeDuration`. In practice we recommend somewhere between 5 minutes and 5 months.
+
+### Limits
```

Comment:
It might be worth writing a Github issue in the repo related to working around this limitation. We can do some interesting things in the wallet automatically like triggering a "bundling" of some group of channels after we reach some threshold. For example if a ledger channels funds `MAX_OUTCOME_ITEMS / 2` outcomes, then for the next one maybe it creates a new "sub ledger" channel funding these application channels, and then funds that sub ledger channel.

Just suggesting we keep track of this general possibility; maybe one day it becomes a priority, or someone interested in the project might want to tackle it.

Reply:
This exact strategy is in my (draft) roadmap 😉

Reply:
Strongly support this: I am minded to not bother with the GH issue if @andrewgordstewart is tracking it elsewhere, though? I feel we suffer a bit from Issue inflation and it might get lost or forgotten about.

Reply:
> I feel we suffer a bit from Issue inflation and it might get lost or forgotten about.

To be a bit pedantic, issue inflation won't cause issues to become lost, but might cause them to be forgotten.

To be a bit more pedantic,
> lost **or** forgotten about

is still true given the first bit of pedantry.